Frontend/Views/Features/Index.cshtml

```diff
@@ -27,86 +25,10 @@
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    Has this transfer started because of an intervention with the academy or trust?
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    @if (!Model.HasTransferReasonBeenSet)
-                    {
-                        <span class="dfe-empty-tag">Empty</span>
-                    }
-                    else
-                    {
-                        @if (Model.IsTransferSubjectToIntervention)
-                        {
-                            <p class="govuk-body">Yes</p>
-                            <p class="govuk-body">@Model.InterventionDetails</p>
-                        }
-                        else
-                        {
-                            <p class="govuk-body">No</p>
-                            <p class="govuk-body">@Model.InterventionDetails</p>
-                        }
-                    }
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a class="govuk-link" asp-action="Reason" asp-route-urn="@Model.Urn" data-test="reason">
-                        Change<span class="govuk-visually-hidden"> has this transfer started because of an intervention with the academy or trust?</span>
-                    </a>
-                </dd>
-            </div>
-            <div class="govuk-summary-list__row">
-                <dt class="govuk-summary-list__key">
-                    What type of transfer is it?
-                </dt>
-                <dd class="govuk-summary-list__value">
-                    @if (Model.TypeOfTransfer == TransferFeatures.TransferTypes.Empty)
-                    {
-                        <span class="dfe-empty-tag">Empty</span>
-                    }
-                    else if (Model.TypeOfTransfer == TransferFeatures.TransferTypes.Other)
-                    {
-                        <p class="govuk-body">Other: @Model.OtherTypeOfTransfer</p>
-                    }
-                    else
-                    {
-                        var displayName = EnumHelpers<TransferFeatures.TransferTypes>.GetDisplayValue(Model.TypeOfTransfer);
-                        @displayName
-                    }
-                </dd>
-                <dd class="govuk-summary-list__actions">
-                    <a class="govuk-link" asp-action="Type" asp-route-urn="@Model.Urn" data-test="type">
-                        Change<span class="govuk-visually-hidden"> what type of transfer is it?</span>
-                    </a>
-                </dd>
-            </div>
-        </dl>
+        <partial for="@Model" name="_FeaturesSummary" />
 
         <form asp-controller="Project" asp-action="Index" asp-route-id="@Model.Urn">
-            <button class="govuk-button">Confirm and continue</button>
```

Comment:
margin-top 6 added here. Default margin bottom 3 (30px) on the summary list = previous 90px set